src/components/Comments/CommentItem.tsx

```diff
@@ -9,13 +8,19 @@ import { Dropdown } from "../../common/Dropdown/Dropdown";
 import { Menu } from "../../common/Menu/Menu";
 import { Block, Elem } from "../../utils/bem";
 import { humanDateDiff, userDisplayName } from "../../utils/utilities";
+import { CommentFormBase } from "./CommentFormBase";
 
 import "./CommentItem.styl";
 import { Button } from "../../common/Button/Button";
 
 
 export const CommentItem: FC<{ comment: any }> = observer(({ comment }) => {
   const resolved = comment.isResolved;
+  const currentUser = window.APP_SETTINGS?.user;
+  const initialComment = comment.text;
+  const [currentComment, setCurrentComment] = useState(initialComment);
+
+  if (comment.isDeleted) return <></>;
```

Comment:
this should just return `null` here from what I can see, any reason for the empty fragment?